.npmignore

```diff
@@ -6,3 +6,9 @@
 !/src/android/**/*
 !/scripts/**/*
```

Comment:
L: If this works, there is no need to change it, but could we remove `!/scripts/**/*` and `/scripts/**/*`, it doesn't seem needed.